packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -0,0 +1,61 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Changing controller updates EditableText', (WidgetTester tester) async {
+    final GlobalKey<EditableTextState> editableTextKey = new GlobalKey<EditableTextState>();
+    final TextEditingController controller1 = new TextEditingController(text: 'Wibble');
+    final TextEditingController controller2 = new TextEditingController(text: 'Wobble');
+    TextEditingController currentController = controller1;
+    StateSetter setState;
+
+    Widget builder() {
+      return new StatefulBuilder(
+        builder: (BuildContext context, StateSetter setter) {
+          setState = setter;
+          return new Directionality(
+            textDirection: TextDirection.ltr,
+            child: new Center(
+              child: new Material(
+                child: new EditableText(
+                  key: editableTextKey,
+                  controller: currentController,
+                  focusNode: new FocusNode(),
+                  style: new Typography(platform: TargetPlatform.android).black.subhead,
+                  cursorColor: Colors.blue,
+                  selectionControls: materialTextSelectionControls,
+                  keyboardType: TextInputType.text,
+                  onChanged: (String value) {},
```

Comment:
trivial nit: style guide says to use `{}` for empty maps and `{ }` for empty code blocks

Reply:
Done.